.circleci/config.yml

```diff
@@ -91,6 +91,6 @@ workflows:
             - test212_jdk11
           filters:
             branches:
-              only: master
+              ignore: /.*/
             tags:
               only: /^v[0-9]+(\.[0-9]+)*$/
```

Comment:
See my comment here https://github.com/zio/zio-actors/pull/28#discussion_r280425328 that needs to be added for each step

Reply:
Done, please re-review.